internal/otelcollector/config/metric/agent/config.go

```diff
@@ -60,7 +60,17 @@ const (
 	MetricGroupTypeVolume    MetricGroupType = "volume"
 )
 
-type K8sClusterMetricsConfig struct {
+type K8sClusterMetricsToDrop struct {
+	*K8sClusterDefaultMetricsToDrop     `yaml:",inline,omitempty"`
+	*K8sClusterPodMetricsToDrop         `yaml:",inline,omitempty"`
+	*K8sClusterContainerMetricsToDrop   `yaml:",inline,omitempty"`
+	*K8sClusterStatefulSetMetricsToDrop `yaml:",inline,omitempty"`
+	*K8sClusterJobMetricsToDrop         `yaml:",inline,omitempty"`
+	*K8sClusterDeploymentMetricsToDrop  `yaml:",inline,omitempty"`
+	*K8sClusterDaemonSetMetricsToDrop   `yaml:",inline,omitempty"`
+}
+
+type K8sClusterDefaultMetricsToDrop struct {
 	// metrics allows enabling/disabling scraped metric.
```

Comment:
This comment is not consistent anymore with the new comments.
Let's replace it with `// Disable some Conatiner metrics by default`
and add a comment above the `K8sNamespacePhase`: `// Disable Namesapce metrics by default`